lib/push_manager.rb

```diff
@@ -184,10 +180,10 @@ def detect_errors_for_commit(commit)
 
     def get_commits_from_push(push)
       git = Git::Git.new(push.branch.repository.name, git_cache_path: GlobalSettings.cache_directory)
-      git.clone_repository(GlobalSettings.jira.ancestor_branches['default'])
+      git.clone_repository(AncestorRef::DEFAULT_ANCESTOR_BRANCH)
```

Comment:
We'll need to change this if/when we ever want to add services outside of just the web repo, probably won't need to be changed right now but it could be helpful to add a comment that points that out

Reply:
good catch, done !